cmd/go-tpc/main.go

```diff
@@ -73,13 +73,15 @@ func openDB() {
 	if err := globalDB.Ping(); err != nil {
 		errString := err.Error()
 		if strings.Contains(errString, unknownDB) {
-			tmpDB, _ = sql.Open(mysqlDriver, ds)
+			tmpDs := fmt.Sprintf("%s:%s@tcp(%s:%d)/", user, password, host, port)
+			tmpDB, _ = sql.Open(mysqlDriver, tmpDs)
 			defer tmpDB.Close()
 			if _, err := tmpDB.Exec(createDBDDL + dbName); err != nil {
 				panic(fmt.Errorf("failed to create database, err %v\n", err))
 			}
 		} else {
 			globalDB = nil
+			panic(fmt.Errorf("failed to connect to database, err %v\n", err))
```

Comment:
This may Introduce an incompatible changes on dump CSV case which in previous don't require the connectivity to the database, but it looks doesn't matter :)

Reply:
For avoiding the incompatibility, removed this line.